### PR TITLE
docs(gen): tutorials requeriments update part ii

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

IAM/Requirements sections updated to match new guidelines.

### CODE REVIEW

File: tutorials/deploy-wordpress-managed-database-mysql8/index.mdx [hunk 1]
```diff
@@ -233,4 +233,4 @@ Now, as WordPress is successfully installed and connections between your users a
 4. The installation of your blog has been successfully. You can log in now and write your first articles or configure the look and feel of WordPress by installing additional themes and modules.
```
Comment (nerda-codes) [inline tutorials/deploy-wordpress-managed-database-mysql8/index.mdx:233]:
```suggestion
4. You can now log in to your blog and write your first articles, or configure the look and feel by installing additional themes and modules.
```

File: tutorials/deploying-a-documentation-website-with-docusaurus-on-scaleway/index.mdx [hunk 2]
```diff
@@ -13,18 +13,17 @@ dates:
...
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have an Object Storage bucket with the [bucket website feature](/storage/object/how-to/use-bucket-website) enabled
-  - You have a [GitHub](https://github.com/) account and [created](https://docs.github.com/en/get-started/quickstart/create-a-repo) an empty repository for your project
-  - You have a local development environment with [Node.js](https://nodejs.org/en/download/) (version 16.14 or above) and `git` installed
-</Message>
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/) with the [bucket website feature](/storage/object/how-to/use-bucket-website) enabled
+- A [GitHub](https://github.com/) account and an [empty repository](https://docs.github.com/en/get-started/quickstart/create-a-repo) for your project
```
Comment (nerda-codes) [inline tutorials/deploying-a-documentation-website-with-docusaurus-on-scaleway/index.mdx:25]:
```suggestion
- A [GitHub](https://github.com/) account and an [empty repository](https://docs.github.com/en/get-started/quickstart/create-a-repo) for your Project
```

File: tutorials/dhcpcd-dedibox/index.mdx [hunk 3]
```diff
@@ -16,24 +16,21 @@ dates:
...
-
+- A Dedibox account logged into the [console](https://console.online.net)
+- A [Dedibox dedicated server] (https://www.scaleway.com/en/dedibox)
+- Root access to the server
 
 ## Installing DHCPCD
 
 1. Change into the root account of your server by using `su` and the password of the `root` user.
-2. For Debian & Ubuntu, update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories: 
+2. For Debian & Ubuntu, update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories:
```
Comment (nerda-codes) [inline tutorials/dhcpcd-dedibox/index.mdx:33]:
```suggestion
2. For Debian and Ubuntu, update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories:
```

File: tutorials/dhcpcd-dedibox/index.mdx [hunk 4]
```diff
@@ -137,7 +134,7 @@ In this tutorial, you will learn how to configure your IPv6 subnet with DHCPCD o
     systemctl status dhcpcd.service
     ```
 
-    You should see an output similar to the following example: 
+    You should see an output similar to the following example:
```
Comment (nerda-codes) [inline tutorials/dhcpcd-dedibox/index.mdx:137]:
```suggestion
    You should see an output similar to the following:
```

File: tutorials/django-ubuntu-focal-fossa/index.mdx [hunk 5]
```diff
@@ -223,6 +222,6 @@ With Django installed, we can now start to create our project and test it on you
 
     The Django project you've created provides the structural basis for designing a more complete site. Check out the Django documentation for more information about how to build your applications and customize your site.
```
Comment (nerda-codes) [inline tutorials/django-ubuntu-focal-fossa/index.mdx:223]:
```suggestion
    The Django project you have created provides the structural basis for designing a more complete website. Check out the Django documentation for more information about how to build your applications and customize your website.
```

File: tutorials/encrypt-s3-data-rclone/index.mdx [hunk 6]
```diff
@@ -23,21 +22,23 @@ Compatible with Windows, macOS X, and various Linux distributions, Rclone addres
...
+- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/)
 
 ## Installing Rclone
 
 You can install `rclone` on your local computer using the pre-built binary files provided. Follow the steps for your computers' operating system:
 
 #### Windows
-On Windows, you can download the latest version of rclone [from their website](https://downloads.rclone.org/rclone-current-windows-amd64.zip).
+
+On Windows, you can download the latest version of Rclone [from their website](https://downloads.rclone.org/rclone-current-windows-amd64.zip).
```
Comment (nerda-codes) [inline tutorials/encrypt-s3-data-rclone/index.mdx:36]:
```suggestion
On Windows, you can [download the latest version of Rclone from the official website](https://downloads.rclone.org/rclone-current-windows-amd64.zip).
```

File: tutorials/erpnext/index.mdx [hunk 7]
```diff
@@ -351,4 +350,4 @@ Our test environment has proven that our ERPNext installation is working, but so
 
 ## Going further
 
-You have now installed, configured, and secured a complete ERPNext 12 application. In the next step, you may want to configure it towards your business needs. For more information regarding the configuration and maintenance of the application, refer to the [official ERPNext handbook](https://docs.erpnext.com/docs/user/manual/en).
+You have now installed, configured, and secured a complete ERPNext 12 application. In the next step, you may want to configure it towards your business needs. For more information regarding the configuration and maintenance of the application, refer to the [official ERPNext handbook](https://docs.erpnext.com/docs/user/manual/en).
```
Comment (nerda-codes) [inline tutorials/erpnext/index.mdx:353]:
```suggestion
You have installed, configured, and secured a complete ERPNext 12 application. In the next step, you may want to configure it towards your business needs. For more information regarding the configuration and maintenance of the application, refer to the [official ERPNext handbook](https://docs.erpnext.com/docs/user/manual/en).
```

File: tutorials/first-steps-linux-command-line/index.mdx [hunk 8]
```diff
@@ -27,18 +27,17 @@ Even if you are using a GUI on your own local machine, you can also choose to us
...
+When you are faced with operating a machine via the command line for the first time, this can be confusing and frustrating. The aim of this document is to is to show you some of the basic principles of using a command line instead of a graphical interface. We will guide you through a series of hands-on exercises to practice some of the main commands you need to get started.
 
 In this document, we cover commands for a **Linux** command line. The commands for a Windows command line are different, so if you are using Windows you should consult [their own documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands). In the context of virtual machines and servers, a Linux-based OS is generally the go-to choice. Unlike Windows, Linux OSes do not require paid-for licenses, they are free and open source.
```
Comment (nerda-codes) [inline tutorials/first-steps-linux-command-line/index.mdx:32]:
```suggestion
In this document, we cover commands for a **Linux** command line. If you are using a Windows command line, we recommend that you [refer to the Windows documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands), as the commands differ. In the context of virtual machines and servers, a Linux-based OS is generally the go-to choice. Unlike Windows, Linux OSes do not require paid-for licenses, they are free and open source.
```

File: tutorials/first-steps-linux-command-line/index.mdx [hunk 9]
```diff
@@ -373,15 +372,15 @@ You can remove files and directories with the `rm` (**r**e**m**ove) command.
 
 When you first install Linux, the only user that exists is **root**, which has inherent "superuser" administrative powers. It is not generally advisable to stay permanently logged in as the superuser, for reasons of security. Instead, you should [create a "normal" user account](#creating-and-logging-into-new-user-accounts-adduser-usermod-and-su) and use that when carrying out operations on your machine.
 
-But as a "normal" user, you will not have permissions for everything, such as updating and upgrading software. In order to carry out this kind of operation, you will need to prefix your command with `sudo` (**su**peruser **do**). 
+But as a "normal" user, you will not have permissions for everything, such as updating and upgrading software. In order to carry out this kind of operation, you will need to prefix your command with `sudo` (**su**peruser **do**).
 
 You are usually prompted to enter a password when you enter a `sudo` command, to ensure that you have the right permissions. Once you have entered your password once, you will not be asked to enter it on subsequent `sudo` commands for a little while.
```
Comment (nerda-codes) [inline tutorials/first-steps-linux-command-line/index.mdx:377]:
```suggestion
You are usually prompted to enter a password when you enter a `sudo` command, to ensure that you have the right permissions. After you have entered your password once, you will not be asked to enter it on subsequent `sudo` commands for a little while.
```

File: tutorials/first-steps-linux-command-line/index.mdx [hunk 10]
```diff
@@ -373,15 +372,15 @@ You can remove files and directories with the `rm` (**r**e**m**ove) command.
...
 
 Carry on to the next section to see how to update and upgrade your system's software, using `sudo`.
 
 ## Updating the system and installing new applications - apt
 
-You can use `apt`, the command for the **A**dvanced **P**ackage **T**ool, to install, update, delete, and manage software packages on Linux systems like Ubuntu and Linux Mint. 
+You can use `apt`, the command for the **A**dvanced **P**ackage **T**ool, to install, update, delete, and manage software packages on Linux systems like Ubuntu and Linux Mint.
 
 By 'package' we are referring essentially to **software**. On Linux, software is usually built as a package, distributed across remote repositories, and managed by users like you via package managers on their local machine.
```
Comment (nerda-codes) [inline tutorials/first-steps-linux-command-line/index.mdx:385]:
```suggestion
In this context, "package" essentially refers to **software**. On Linux, software is usually built as a package, distributed across remote repositories, and managed by users like you via package managers on their local machine.
```

File: tutorials/flask/index.mdx [hunk 11]
```diff
@@ -38,7 +37,7 @@ If you do not have Python installed on your computer, download the
 installer from the Python [official website](https://www.python.org/downloads/).
 
 To make sure your Python installation is functional, you can open a
-terminal window and type python3, or if that does not work, just python.
+terminal window and type _python3_, or if that does not work, just _python_.
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:40]:
```suggestion
terminal window and type `python3`, or if that does not work, just `python`.
```

File: tutorials/flask/index.mdx [hunk 12]
```diff
@@ -52,7 +51,7 @@ Python 3.8.10 (default, Nov 26 2021, 20:14:08)
 Type "help", "copyright", "credits" or "license" for more information.
 ```
 
-To exit the interactive prompt, you can type exit() and press Enter. On
+To exit the interactive prompt, you can type _exit()_ and press Enter. On
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:54]:
```suggestion
To exit the interactive prompt, you can type `exit()` and press Enter. On
```

File: tutorials/flask/index.mdx [hunk 13]
```diff
@@ -52,7 +51,7 @@ Python 3.8.10 (default, Nov 26 2021, 20:14:08)
...
 the Linux and Mac OS X versions of Python you can also exit the
 interpreter by pressing Ctrl-D. On Windows, the exit shortcut is Ctrl-Z
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:56]:
```suggestion
interpreter by pressing `Ctrl+D`. On Windows, the exit shortcut is `Ctrl+Z`,
```

File: tutorials/flask/index.mdx [hunk 14]
```diff
@@ -64,11 +63,11 @@ from where anybody can download and install them. The official
 Python package repository is called PyPI, which stands for Python
 Package Index.
 
-1. (Optional) If you do not have `pip` installed, launch
+1. Optionally, if you do not have `pip` installed, launch:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:66]:
```suggestion
1. Optionally, if you have not installed `pip`, run the following command:
```

File: tutorials/flask/index.mdx [hunk 15]
```diff
@@ -152,7 +151,7 @@ We will create several URLs routes:
         app.run(host='0.0.0.0')
     ```
 
-2. Restart the application using
+2. Restart the application using:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:154]:
```suggestion
2. Restart the application.
```

File: tutorials/flask/index.mdx [hunk 16]
```diff
@@ -193,15 +192,15 @@ if __name__ == "__main__":
     app.run(host='0.0.0.0', port=80)
 ```
 
-1. Create a directory called templates
+1. Create a directory called templates:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:195]:
```suggestion
1. Create a directory called `templates`:
```

File: tutorials/flask/index.mdx [hunk 17]
```diff
@@ -193,15 +192,15 @@ if __name__ == "__main__":
...
     ```
     mkdir templates
     ```
-2. Open a `test.html` file and paste the following
+2. Open a `test.html` file and paste the following:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:199]:
```suggestion
2. Create a `test.html` file:
```

File: tutorials/flask/index.mdx [hunk 18]
```diff
@@ -193,15 +192,15 @@ if __name__ == "__main__":
...
     ```
     nano test.html
     ```
-3. Paste the following
+3. Paste the following:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:203]:
```suggestion
3. Paste the following code into your `test.html` file, save your changes, and exit.
```

File: tutorials/flask/index.mdx [hunk 19]
```diff
@@ -215,7 +214,7 @@ if __name__ == "__main__":
     </div>
     {% endblock %}
     ```
-4. Open a `layout.html` file and paste the following
+4. Open a `layout.html` file and paste the following:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:217]:
```suggestion
4. Create a `layout.html` file using `nano layout.html`, and paste the following code:
```

File: tutorials/flask/index.mdx [hunk 20]
```diff
@@ -250,7 +249,7 @@ if __name__ == "__main__":
 
 Let's try and display random Scaleway catch phrase instead of always the same one. We will need to pass both the _name variable_ and the _quote variable_.
 
-1. In the `templates` directory, edit the `test.html` to match the following
+1. In the `templates` directory, edit the `test.html` to match the following:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:252]:
```suggestion
1. In the `templates` directory, edit `test.html` to match the following:
```

File: tutorials/flask/index.mdx [hunk 21]
```diff
@@ -261,7 +260,7 @@ Let's try and display random Scaleway catch phrase instead of always the same on
     </div>
     {% endblock %}
     ```
-2. Save and exit
+2. Save and exit.
 3. In the application called `hello.py` update the configuration to look like this:
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:264]:
```suggestion
3. In the application called `hello.py`, update the configuration to look like the following:
```

File: tutorials/flask/index.mdx [hunk 22]
```diff
@@ -296,7 +295,7 @@ Let's try and display random Scaleway catch phrase instead of always the same on
     As you can notice, we updated the `quotes` variable with an array of multiples quotes. These can be accessed as quote[0], quote[1], quote[2] and so on. The function **randint()** returns a random number between 0 and the total number of quotes, one is subtracted because we start counting from zero.
 
     We also added a **locals()** function which always returns a dictionary of the current namespace.
-4. Save and exit
+4. Save and exit.
 5. Run the application `hello.py`. It will return one of these quotes at random.
```
Comment (nerda-codes) [inline tutorials/flask/index.mdx:299]:
```suggestion
5. Run the application `hello.py`.
```

File: tutorials/foreman-puppet/index.mdx [hunk 23]
```diff
@@ -141,23 +143,23 @@ You can log in to Foreman using the credentials that were returned once it was i
 
 <Lightbox src="scaleway-foreman_login.webp" alt="" />
 
-## Foreman Dashboard
+## Foreman dashboard
 
 After login, the Foreman dashboard displays an overview of your Puppet environment.
 
 It displays a summary of "Host Configuration Status", showing the number of Puppet agent nodes and their respective statuses. At the first connection, one single Instance should be visible in the "good host" section.
```
Comment (nerda-codes) [inline tutorials/foreman-puppet/index.mdx:150]:
```suggestion
A summary of "Host Configuration Status" displays, with the amount of Puppet agent nodes and their statuses. Upon first connection, one single Instance should be visible in the "good host" section.
```

File: tutorials/get-started-deploy-kapsule/index.mdx [hunk 24]
```diff
@@ -23,17 +23,16 @@ Future videos will cover topics like load balancing and storage for your Kuberne
...
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+- Followed the [first tutorial](/tutorials/get-started-containers-docker/) in this series, and pushed the `whoami` image to a Container Registry
 
 ## Why do we need Kubernetes?
 
 In our previous tutorial, we saw how to containerize an application. We achieved this by using Docker, an open source platform for packaging applications into containers. We created and ran our containerized application on our local machine, and then pushed the container image to a Container Registry.
 
-While manually running and managing one container image on one local machine is fine, in a production environment we might need huge amounts of containers running simultaneously on multiple machines. It is very difficult to manage this by hand, and that is where Kubernetes can help us!
+While manually running and managing one container image on one local machine is fine, in a production environment we might need huge amounts of containers running simultaneously on multiple machines. It is very difficult to manage this by hand, and that is where Kubernetes can help us.
```
Comment (nerda-codes) [inline tutorials/get-started-deploy-kapsule/index.mdx:35]:
```suggestion
While manually running and managing one container image on one local machine is fine, in a production environment we might need huge amounts of containers running simultaneously on multiple machines. This is difficult to manage manually, and that is where Kubernetes can help us.
```

File: tutorials/glusterfs/index.mdx [hunk 25]
```diff
@@ -94,14 +92,14 @@ sudo add-apt-repository ppa:gluster/glusterfs-9
     ```
     apt install glusterfs-server -y
     ```
-2. Start the glusterd service and enable it to launch every time at system boot
+2. Start the glusterd service and enable it to launch every time at system boot:
```
Comment (nerda-codes) [inline tutorials/glusterfs/index.mdx:95]:
```suggestion
2. Start the glusterd service, and enable it to launch at every system boot:
```

File: tutorials/glusterfs/index.mdx [hunk 26]
```diff
@@ -211,11 +209,11 @@ In this step, we will mount the `glusterfs` volume `vol01` to the Ubuntu client,
       To mount glusterfs permanently to the Ubuntu client system, we can add the volume to the `/etc/fstab`.
     </Message>
 5. Edit the `/etc/fstab` configuration file: `vim /etc/fstab`.
-6. Paste the following configuring:`gluster01:/vol01 /mnt/glusterfs glusterfs defaults,_netdev 0 0`.
+6. Paste the following configuring: `gluster01:/vol01 /mnt/glusterfs glusterfs defaults,_netdev 0 0`.
```
Comment (nerda-codes) [inline tutorials/glusterfs/index.mdx:212]:
```suggestion
6. Paste the following configuration: `gluster01:/vol01 /mnt/glusterfs glusterfs defaults,_netdev 0 0`.
```

File: tutorials/gpu-super-resolution/index.mdx [hunk 27]
```diff
@@ -24,14 +24,14 @@ After the model has been trained, a separate test set should be used to guarante
...
-  - You have a [GPU instance](https://www.scaleway.com/en/gpu-3070-instances/) running on UbuntuML
-  - You have sudo privileges or access to the root user
-  - You have [configured Jupyter Notebook on your GPU instance](/tutorials/setup-jupyter-notebook/)
-</Message>
 
-## Preparing the Data
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [GPU Instance](/compute/instances/how-to/create-an-instance) running on UbuntuML
+- Configured [Jupyter Notebook](/tutorials/setup-jupyter-notebook/) on your GPU instance
```
Comment (nerda-codes) [inline tutorials/gpu-super-resolution/index.mdx:31]:
```suggestion
- Configured [Jupyter Notebook](/tutorials/setup-jupyter-notebook/) on your GPU Instance
```

File: tutorials/hadoop/index.mdx [hunk 28]
```diff
@@ -181,23 +179,23 @@ MapReduce is a programming framework that allows us to perform distributed and p
 
 MapReduce consists of two distinct tasks – Map and Reduce. As the name MapReduce suggests, reducer phase takes place after mapper phase has been completed.
 
-1. Create a directory called `input` in our home directory
+1. Create a directory called `input` in our home directory:
```
Comment (nerda-codes) [inline tutorials/hadoop/index.mdx:182]:
```suggestion
1. Create a directory called `input` in your home directory:
```

File: tutorials/hadoop/index.mdx [hunk 29]
```diff
@@ -181,23 +179,23 @@ MapReduce is a programming framework that allows us to perform distributed and p
...
     ```
     wget  https://jar-download.com/cache_jars/org.apache.hadoop/hadoop-mapreduce-examples/3.0.3/jar_files.zip
     ```
-4. Unzip the file
+4. Unzip the file:
     ```
     unzip jar_files.zip
     ```
-5. Move the `jar_files` to `/usr/local/hadoop/share/hadoop/mapreduce/hadoop/`
+5. Move the `jar_files` to `/usr/local/hadoop/share/hadoop/mapreduce/hadoop/`.
```
Comment (nerda-codes) [inline tutorials/hadoop/index.mdx:198]:
```suggestion
5. Move `jar_files` to `/usr/local/hadoop/share/hadoop/mapreduce/hadoop/`.
```

File: tutorials/hadoop/index.mdx [hunk 30]
```diff
@@ -181,23 +179,23 @@ MapReduce is a programming framework that allows us to perform distributed and p
...
     ```
     mv hadoop-mapreduce-examples-3.0.3.jar /usr/local/hadoop/share/hadoop/mapreduce/.
```
Comment (nerda-codes) [inline tutorials/hadoop/index.mdx:200]: Logically, this step should be "mv jar_files /usr/local/hadoop/share/hadoop/mapreduce/" but I don't really know for sure

Comment (nerda-codes) [inline tutorials/hadoop/index.mdx:200]:
```suggestion
    mv hadoop-mapreduce-examples-3.0.3.jar /usr/local/hadoop/share/hadoop/mapreduce/
```

File: tutorials/install-apache-ansible/index.mdx [hunk 31]
```diff
@@ -27,15 +29,15 @@ Once Ansible is installed, we need to specify Ansible which hosts to talk to. We
...
     ```
     mkdir ansible-apache
     ```
-2. Move to this newly created directory
+2. Move to this newly created directory:
     ```
     cd ~/ansible-apache/
     ```
-3. Create a new file `ansible.cfg`
+3. Create a new file `ansible.cfg`:
```
Comment (nerda-codes) [inline tutorials/install-apache-ansible/index.mdx:40]:
```suggestion
3. Create a new `ansible.cfg` file:
```

File: tutorials/install-apache-ansible/index.mdx [hunk 32]
```diff
@@ -157,7 +159,7 @@ The apache2_module module takes two actions:
 - **name:** The name of the module to enable, such as write.
 - **attribute:** Either present or away, being on if the module needs to be enabled or not.
 
-1. Open apache.yml for editing
+1. Open apache.yml for editing:
```
Comment (nerda-codes) [inline tutorials/install-apache-ansible/index.mdx:162]:
```suggestion
1. Open `apache.yml` to edit it:
```

File: tutorials/install-configure-couchdb/index.mdx [hunk 33]
```diff
@@ -301,4 +300,4 @@ To secure it, it is recommended to install a firewall on each node to restrict c
 
 Connections from the internet to your CouchDB cluster are now limited. As an additional security measure, you can disable the public IP address of the nodes as they communicate via the internal IP with each other.
 
-For more information how to configure and use CouchDB for your applications, refer to the [official documentation](https://docs.couchdb.org/en/latest/).
+For more information how to configure and use CouchDB for your applications, refer to the [official documentation](https://docs.couchdb.org/en/latest/).
```
Comment (nerda-codes) [inline tutorials/install-configure-couchdb/index.mdx:303]:
```suggestion
For more information on how to configure and use CouchDB for your applications, refer to the [official documentation](https://docs.couchdb.org/en/latest/).
```

File: tutorials/install-ispconfig/index.mdx [hunk 34]
```diff
@@ -18,14 +18,13 @@ ISPConfig is an open source, transparent, free, stable and secure administration
...
-  - You have [created your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance/) that runs Ubuntu (for performance reasons, it is recommended to use an Instance with at least 4 GB of RAM)
-  - You have a domain or subdomain pointed to your Instance
-  - You have set the hostname and [reverse DNS](/compute/instances/how-to/configure-reverse-dns/) of your Instance to a valid FQDN
-</Message>
-
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
+- An [Instance](/compute/instances/how-to/create-an-instance) running on buntu
```
Comment (nerda-codes) [inline tutorials/install-ispconfig/index.mdx:24]:
```suggestion
- An [Instance](/compute/instances/how-to/create-an-instance) running on Ubuntu
```

File: tutorials/install-medusa/index.mdx [hunk 35]
```diff
@@ -77,8 +76,9 @@ Inside the project's root directory, specified at the start of the installation
 ```
 
 You can edit the files to adapt them towards your requirements. For instance, if you wish to make changes to the homepage of your MedusaJS application, you can modify the `my-medusa-store/storefront/src/pages/index.tsx` file.
+
 This particular file comprises the components used for rendering your application's homepage. If you want to make changes to the backend of your application, you can edit the files in the `my-medusa-store/backend` towards your specific requirements.
```
Comment (nerda-codes) [inline tutorials/install-medusa/index.mdx:80]:
```suggestion
This particular file comprises of the components used for rendering your application's homepage. If you want to make changes to the backend of your application, you can edit the files in the `my-medusa-store/backend` towards your specific requirements.
```

File: tutorials/install-parse-server/index.mdx [hunk 36]
```diff
@@ -312,4 +311,4 @@ Parse server comes with a Dashboard for managing your Parse server applications.
 1. Access the Parse server dashboard by visiting the URL `http://your-server-ip:4040`
     <Lightbox src="scaleway-parse_dashboardlogin.webp" alt="" />
 2. Add the credentials that you entered in the `parse-darshboard-config.json` file.
-    <Lightbox src="scaleway-parse_dashboard.webp" alt="" />
+    <Lightbox src="scaleway-parse_dashboard.webp" alt="" />
```
Comment (nerda-codes) [inline tutorials/install-parse-server/index.mdx:314]: Are you sure about the image name?

File: tutorials/install-php-composer/index.mdx [hunk 37]
```diff
@@ -18,12 +18,11 @@ PHP [Composer](https://getcomposer.org/) is a package dependency management tool
...
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
-  - You have PHP 7.4 or higher installed on your Instance
-</Message>
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
+- An [Instance](/compute/instances/how-to/create-an-instance) running on Ubuntu Focal Fossa
+- PHP 7.4 or higher installed on your Instance
```
Comment (nerda-codes) [inline tutorials/install-php-composer/index.mdx:25]:
```suggestion
- Installed PHP 7.4 or higher on your Instance
```

File: tutorials/installation-jekyll/index.mdx [hunk 38]
```diff
@@ -82,24 +80,24 @@ Jekyll provides a basic web server for testing during development, but its main
 
 We use [Nginx](https://nginx.org/) in our example on the production server.
 
-1. Start by installing the required software
+1. Start by installing the required software:
     ```
     sudo apt-get install ruby ruby-dev nginx make build-essential git-core
     ```
-2. Run Git on the production server, therefore you have to create 'git' user
+2. Run Git on the production server, therefore you have to create 'git' user:
```
Comment (nerda-codes) [inline tutorials/installation-jekyll/index.mdx:87]:
```suggestion
2. Run Git on the production server, therefore you have to create a user called `git`:
```

File: tutorials/installation-lemp-ubuntu-focal/index.mdx [hunk 39]
```diff
@@ -241,4 +240,4 @@ Let's Encrypt provides a [certbot](https://certbot.eff.org/) to configure Nginx
     ```
     ufw allow 'Nginx HTTPS'
     ```
-7. Access your site with HTTPS: `https://example.com/`.
+7. Access your site with HTTPS: `https://example.com/`.
```
Comment (nerda-codes) [inline tutorials/installation-lemp-ubuntu-focal/index.mdx:243]:
```suggestion
7. Access your website with HTTPS: `https://example.com/`.
```

File: tutorials/install-parse-server/index.mdx (hunk 36 above)
Comment (jcirinosclwy) [reply tutorials/install-parse-server/index.mdx:314]: Idk where this came from. But the image exists and is currently featured in the tutorial, so I guess I'll leave it here?

File: tutorials/hadoop/index.mdx (hunk 30 above)
Comment (jcirinosclwy) [reply tutorials/hadoop/index.mdx:200]: I'll trust you on this one, cause I have no idea 😅

**Review by nerda-codes — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**